index.php

```diff
@@ -11,21 +11,23 @@ function table_rows() {
   );
 
   foreach ($paths as $path => $name) {
-    $databases = array();
-    $files = array();
-
-    exec("ls $path/sanitized/*.sql.gz", $databases);
+    $dbs = $dbs_civi = $files = array();
+    exec("ls $path/sanitized/*.sql.gz", $dbs);
+    exec("ls $path/sanitized_civi/*.sql.gz", $dbs_civi);
     exec("ls $path/files_backups/*.tar.gz", $files);
-    foreach (array_reverse($databases) as $id => $database) {
-      create_row($name, $database, array_reverse($files)[$id]);
+    $r_dbs = array_reverse($dbs);
+    $r_dbs_civi = array_reverse($dbs_civi);
+    $r_files = array_reverse($files);
+    foreach ($r_dbs as $id => $db) {
+      create_row($name, $db, (isset($r_dbs_civi[$id]) ? $r_dbs_civi[$id] : ''), $r_files[$id]);
     }
   }
 }
 
 /**
  * Print the HTML for a single table row.
  */
-function create_row($name, $db_path, $file_path) {
+function create_row($name, $db_path, $db_civi_path, $file_path) {
```

Comment:
can we make the new argument the last one? We won't have civi for most of our sites, and it would be easier not to need to pass NULL when backing up the docs site (for example).

Reply:
Sure, happy to swap the arguments (see revised PR). I will note that when we pass `''` (not `NULL`), that happens automatically (or not) based on whether there's a CiviCRM db backup in place, so that this works currently for all four properties. 

And then if we decide to add CiviCRM to, for example, the events site (to do event registrations), all we'd have to do, backup/sanitation-wise, is add the `*_CIVI` settings to the `config.ini` file on events.b.org, use the current `sql-dump-sanitize.php` script), and add the symlink to the `sanitized_civi` directory, and this function will start showing the Civi backups automatically as they start to be created.